src/assets/scripts/client/commands/parsers/CommandParser.js

```diff
@@ -128,7 +128,7 @@ export default class CommandParser {
      */
     _extractCommandsAndArgs(rawCommandWithArgs) {
         const commandOrCallsignIndex = 0;
-        const commandArgSegmentsWithCallsign = rawCommandWithArgs.split(COMMAND_ARGS_SEPARATOR);
+        const commandArgSegmentsWithCallsign = rawCommandWithArgs.split(COMMAND_ARGS_SEPARATOR).filter(t => t);
```

Comment:
What exactly are we `.filter(t => t)` here? 

Oh, I think I see it. An empty string will evaluate as falsy here and will not be returned. Is that correct? If so, that's clever.

Reply:
Yes, that's pretty much it.